File: lib-model/src/main/java/com/minesec/msa/client/sdk/ResultDto.java
package com.minesec.msa.client.sdk;

import lombok.Setter;

/**
 * @author eric.song
 * @since 2022/12/14 11:54
 */
@Setter
public class ResultDto {
    private int code;
    private String msg;

    public boolean approved() {
        return (0 == this.code);
    }

    public String toErrorMessage() {
        return this.code + ":" + this.msg;
    }

    public String getRespCode() {
        return "" + this.code;
    }

    public String getRespText() {
        return this.msg;
    }
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/ResultDto_.kt
package com.minesec.msa.client.sdk.management.model_k

/**
 * @author eric.song
 * @since 2024/6/18 14:18
 */
open class ResultDto_(var code: Int, var msg: String) {
    fun approved(): Boolean {
        return code == 0
    }

    fun toErrorMessage(): String {
        return "$code: $msg"
    }

    fun getRespCode(): String {
        return "$code"
    }

    fun getRespText(): String {
        return msg
    }
}